packages/expo-blur/jest.config.js

```diff
@@ -0,0 +1,11 @@
+const withEnzyme = require('jest-expo-enzyme');
```

Comment:
This is using `expo-module-scripts`, just with `jest-expo-enzyme` instead of `jest-expo`. Maybe we could make `withEnzyme` recursive but that could get messy

Reply:
OK. let's open an issue to make expo-module-scripts use jest-expo-enzyme once we've vetted it in a few modules. expo-module-scripts was designed to let you provide custom configs for short-term purposes (like here) but ultimately we need all packages to be streamlined so we can update them in concert.